daily-builds.md

```diff
@@ -1,23 +1,23 @@
-# .NET Core Daily Builds -- Servicing and future releases
+# .NET Daily Builds -- Servicing and future releases
 
-The .NET Core team builds many branches of the product daily. You can install these builds to try out functionality prior to release.
+The .NET team builds many branches of the product daily. You can install these builds to try out functionality prior to release.
 
 ## Future Releases
```

Comment:
Thoughts on referencing the [.NET (Preview) docker images](https://hub.docker.com/_/microsoft-dotnet-nightly/) here?

Reply:
Good call. Updated. What do you think of the wording?

Reply:
Looks good to me.